alerts/keys-api-outdated-validators.rule.yml

```
@@ -2,11 +2,11 @@ groups:
   - name: Keys API. Validators
     rules:
       - alert: KeysApiOutdatedValidators
-        expr: validators_registry_enabled{} == 1 AND (time() - lido_keys_api_validators_registry_last_update_block_timestamp{} >= 1800)
+        expr: validators_registry_enabled{} == 1 AND (time() - lido_keys_api_validators_registry_last_update_block_timestamp{} >= 3600)
         labels:
           severity: critical
           service: keys_api
           app_team: tooling
         annotations:
           summary: Validators are outdated
-          description: Validators were not updated for more than 30 minutes
+          description: 'Validators were not updated for more than  {{ with query "time() - lido_keys_api_validators_registry_last_update_block_timestamp{}" }}{{ . | first | value | humanizeDuration }}{{ end }} minutes'
```

```suggestion
          description: 'Validators were not updated for more than {{ with query "time() - lido_keys_api_validators_registry_last_update_block_timestamp{}" }}{{ . | first | value | humanizeDuration }}{{ end }}'
```
- remove extra space
- `minutes` are not needed, `humanizeDuration` adds the necessary suffixes by itself